src/data.js

```diff
@@ -0,0 +1,48 @@
+class Account {
+    constructor(name) {
+        this.name = name;
+        this.transactions = [];
+    }
+
+    get balance() {
+        return this.transactions.reduce((acc, transaction) => {
+            if (transaction.dstAccount === this) {

```

Comment:
`return transaction.dstAccount === this ? ... : ...` would be nicer here, since the if just contains single line returns.

Even further, you could extract the condition into a variable (for readability) and reduce it all the way to `return acc + (isCredit ? 1 : -1) * transaction.amount`.